backend/codesirius/problems/views/tag.py

```
@@ -13,128 +11,88 @@
-    def get(self, request: Request, pk: int) -> CodesiriusAPIResponse:
-        """
-        Get a tag by its primary key
-        """
+class TagRetrieveUpdateDestroyAPIView(generics.RetrieveUpdateDestroyAPIView):
+    """
+    API to retrieve, update, or delete a tag by its primary key
+    """
+
+    queryset = Tag.objects.all()
+    serializer_class = TagSerializer
+    permission_classes = [IsAdminOrReadOnly]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)